provider/aws/sia-ec2/cmd/siad/main.go

```diff
@@ -52,20 +55,43 @@ func main() {
 	ztsServerName := flag.String("ztsservername", "", "zts server name for tls connections")
 	ztsCACert := flag.String("ztscacert", "", "zts CA certificate file")
 	ztsAwsDomain := flag.String("ztsawsdomain", "", "ZTS AWS Domain")
+	providrPntDomain := flag.String("providerParentDomain", "athenz", "providerParentDomain")
+	version := flag.String("version", "1.0", "version")
```

Comment:
this doesn't look right. the caller should not be able to specify their version number otherwise anyone can use any value they want. This should be hardcoded value with the capability to specify it during the build process.

Reply:
updated